hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/container/placement/algorithms/TestSCMContainerPlacementRackAware.java

```diff
@@ -107,24 +107,23 @@ private void setup(int datanodeCount) {
       DatanodeDetails datanodeDetails =
           MockDatanodeDetails.createDatanodeDetails(
           hostname + i, rack + (i / NODE_PER_RACK));
+      datanodes.add(datanodeDetails);
+      cluster.add(datanodeDetails);
       DatanodeInfo datanodeInfo = new DatanodeInfo(
-          datanodeDetails, NodeStatus.inServiceHealthy(),
-          UpgradeUtils.defaultLayoutVersionProto());
+              datanodeDetails, NodeStatus.inServiceHealthy(),
```

Comment:
Can you check the indentation on these lines - it looks wrong, although checkstyle has passed it. I think most of the lines changed from here to the bottom of the PR are all indented more than they need to be.

BTW, you can run checkstyle locally to see if the style is correct if you change the indentation here. From the root of the repo run:

```
./hadoop-ozone/dev-support/checks/checkstyle.sh
```

Its the same in TestSCMContainerPlacementRackScatter.java too.

Reply:
Yeah might have been a typo. This testcase was failing was debugging it.